### ISSUE TITLE
7.5 On Print Method Alternative for Fun

### ISSUE THREAD

**Issue by MayaGans** [JohnCoene/javascript-for-r-comments#27]
Not sure this is worth including but figured I'd share -- after the chapter 7 I was playing around with including the style as an argument and this code works too instead of piping which I thought was a cool alternative

Maybe if there was like an exercise or challenge section at the end of the chapter this could be a cool way of synthesizing what we already learned? 

```
gio <- function(data, width = NULL, height = NULL,
                elementId = NULL, style = c("magic", "juicyCake", "blueInk", "earlySpring", "sunset")) {

  style = match.arg(style)
  # forward options using x
  x = list(
    data = data,
    style = style
  )
```

Feel free to close this, just thought I'd share 😄

**Comment by JohnCoene** [comment]
Yes of course but I think I have a few words on that later in the book.

Piping because nearly indispensable as you as such arguments: gio.js has 100s of options which is going to make for a weird function :p

But let me know your thoughts when you reach that point :)
